integration-cli/docker_utils.go

```diff
@@ -378,16 +394,66 @@ func getAllContainers() (string, error) {
 	return out, err
 }
 
-func deleteAllContainers() error {
-	containers, err := getAllContainers()
+type cont struct {
+	Id string
+}
+
+func allConts() ([]cont, error) {
```

Comment:
Because `sockRequest` now failing if status is not `200` and `rm` returns `204`. I prefer to use my `httpClient` to implement `sockRequest`.